students/boksil/users/views.py

```diff
@@ -39,3 +41,34 @@ def post(self, request):
         
         except IntegrityError:
             return JsonResponse({'massage': 'KEY_ERROR_EMAIL_EXIST'}, status=400)
+
+
+class SigninView(View):
+    def post(self, request):
+        try:
+            data           = json.loads(request.body)
+            input_email    = data['email']
+            input_password = data['password'].encode('utf-8')
+
+            if input_email == '' or input_password == '':
+                return JsonResponse({'message': 'INVALID_USER'}, status=401)
+
+            db_email    = User.objects.get(email=input_email)
+            db_password = db_email.password # 가져온 이메일의 패스워드
+            db_password = db_password.encode('utf-8') # 패스워드를 다시 encode
```

Comment:
이정도는 변수 하나로 한번에 처리해주셔도 됩니다.
```suggestion
            db_password = db_email.password.encode('utf-8')
```

Reply:
수정했습니다!